sdks/go/pkg/beam/model/fnexecution_v1/beam_provision_api.pb.go

```diff
@@ -197,25 +305,35 @@ var _ProvisionService_serviceDesc = grpc.ServiceDesc{
-	0x5a, 0x5d, 0x2b, 0x16, 0x41, 0xbb, 0x51, 0x69, 0x77, 0x6e, 0x42, 0x32, 0x73, 0xa7, 0xa6, 0x74,
-	0x72, 0xe3, 0x24, 0x33, 0xf8, 0x02, 0x3e, 0x81, 0x2f, 0xe1, 0x63, 0x4a, 0x33, 0x4c, 0x69, 0x95,
-	0x42, 0x71, 0x9b, 0x73, 0xcf, 0x97, 0x73, 0x0e, 0x09, 0x25, 0x88, 0x94, 0x9b, 0x0c, 0x0b, 0x65,
-	0x15, 0x6a, 0x2e, 0x8c, 0x62, 0x26, 0x43, 0x87, 0xf4, 0x02, 0xb3, 0x31, 0x13, 0x46, 0x44, 0x6f,
-	0xc0, 0x66, 0x47, 0x2c, 0xc5, 0x18, 0xa6, 0x2c, 0xd1, 0x1c, 0x3e, 0x20, 0xca, 0x9d, 0x42, 0xcd,
-	0x8a, 0x6e, 0xeb, 0x74, 0x8c, 0x38, 0x9e, 0x42, 0xc7, 0x9b, 0x64, 0x9e, 0x74, 0xac, 0xcb, 0xf2,
-	0xc8, 0x95, 0x90, 0x76, 0x93, 0x34, 0x1e, 0xc0, 0xbd, 0x54, 0xf8, 0x81, 0x4e, 0x70, 0x08, 0xef,
-	0x39, 0x58, 0xd7, 0x8e, 0x49, 0xf8, 0x57, 0xb2, 0x06, 0xb5, 0x05, 0xfa, 0x48, 0xb6, 0x94, 0x4e,
-	0x30, 0x0c, 0xce, 0x82, 0xcb, 0xfd, 0xde, 0x35, 0x5b, 0x2b, 0x0a, 0x5b, 0x66, 0x79, 0x42, 0xfb,
-	0x33, 0x20, 0x87, 0x4b, 0xef, 0xf4, 0x84, 0xd4, 0x26, 0x28, 0xb9, 0x8a, 0x3d, 0x7d, 0x6f, 0xb8,
-	0x3d, 0x41, 0x39, 0x88, 0x69, 0x93, 0xec, 0xce, 0x9e, 0xb5, 0x48, 0x21, 0xdc, 0xf0, 0xc2, 0xce,
-	0x04, 0xe5, 0x93, 0x48, 0x81, 0xf6, 0x49, 0xdd, 0x28, 0x03, 0x53, 0xa5, 0x81, 0xa3, 0x99, 0xfd,
-	0x66, 0xc3, 0x4d, 0x9f, 0xac, 0xc1, 0xca, 0xf6, 0xac, 0x6a, 0xcf, 0x46, 0xbe, 0xfd, 0xf0, 0xb8,
-	0x32, 0x3c, 0x97, 0xf7, 0xbd, 0xef, 0x80, 0xd4, 0xe7, 0x39, 0x46, 0x90, 0x15, 0x2a, 0x02, 0xfa,
-	0x15, 0x90, 0xfa, 0xef, 0x0d, 0xe8, 0xcd, 0x9a, 0x6d, 0x57, 0xec, 0xda, 0xba, 0xfd, 0xb7, 0xbf,
-	0x1c, 0xbf, 0x7f, 0x4f, 0xce, 0x57, 0x11, 0x16, 0x01, 0xfd, 0x83, 0xb9, 0xfd, 0xce, 0xa8, 0xd7,
-	0xa3, 0x05, 0x95, 0x17, 0x5d, 0x59, 0xf3, 0x93, 0x5c, 0xfd, 0x04, 0x00, 0x00, 0xff, 0xff, 0x2e,
-	0xae, 0x1f, 0x23, 0x61, 0x02, 0x00, 0x00,
+	// 469 bytes of a gzipped FileDescriptorProto
+	0x1f, 0x8b, 0x08, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0xff, 0x9c, 0x93, 0xcf, 0x6e, 0xd3, 0x40,
+	0x10, 0xc6, 0xe5, 0xc6, 0x18, 0x3a, 0x40, 0x1b, 0xad, 0x80, 0xba, 0x56, 0x91, 0x50, 0x04, 0x12,
+	0xa7, 0x2d, 0x2d, 0x20, 0x6e, 0x20, 0xd2, 0x0a, 0xa8, 0x04, 0xb4, 0xda, 0x9e, 0xe0, 0x62, 0xf9,
+	0xcf, 0x24, 0xdd, 0x34, 0xde, 0x59, 0x76, 0xd7, 0x11, 0xf0, 0x1a, 0xbc, 0x04, 0x8f, 0xc5, 0x89,
+	0xe7, 0x40, 0x5e, 0x3b, 0xa1, 0x05, 0x2a, 0xa5, 0xdc, 0xec, 0xf9, 0xf6, 0xfb, 0x79, 0xbf, 0xf1,
+	0x0c, 0xc4, 0x39, 0x66, 0x55, 0xaa, 0x0d, 0xcd, 0xa4, 0x95, 0xa4, 0xd2, 0x4c, 0x4b, 0xae, 0x0d,
+	0x39, 0x62, 0x0f, 0xc8, 0x8c, 0x79, 0xa6, 0xb3, 0xe2, 0x04, 0x79, 0x73, 0x88, 0x57, 0x54, 0xe2,
+	0x94, 0x8f, 0x54, 0x8a, 0x9f, 0xb1, 0xa8, 0x9d, 0x24, 0xc5, 0x67, 0x3b, 0xc9, 0xd6, 0x98, 0x68,
+	0x3c, 0xc5, 0x6d, 0x6f, 0xca, 0xeb, 0xd1, 0xb6, 0x75, 0xa6, 0x2e, 0x5c, 0x0b, 0x19, 0x6c, 0xc2,
+	0xc6, 0x6b, 0x74, 0x47, 0x73, 0xfc, 0x81, 0x1a, 0x91, 0xc0, 0x4f, 0x35, 0x5a, 0x37, 0x28, 0x21,
+	0xfe, 0x5b, 0xb2, 0x9a, 0x94, 0x45, 0xf6, 0x06, 0x42, 0xa9, 0x46, 0x14, 0x07, 0xf7, 0x82, 0x87,
+	0xd7, 0x77, 0x9f, 0xf0, 0xa5, 0xae, 0xc2, 0xcf, 0xb3, 0x3c, 0x61, 0xf0, 0x23, 0x80, 0x9b, 0xe7,
```

Comment:
Do you mean `0xf7, 0x77, 0x9f, 0xf0, 0xa5, 0xae, 0xe2, 0xcf, 0xb3, 0x3c, 0x61, 0xf0, 0x23, 0x80, 0x9b, 0xe7`? j/k